projects/app/js/restore.test.js

```
@@ -0,0 +1,508 @@
+}));
+
+// Import mocked modules
+const { dbClear, dbAddMultiple, dbPut } = await import('../shared/js/db.js');
+const {
+    validateCategorySchema,
+    validateHistorySchema,
+    validateSettingsSchema,
+    validateAlarmSchema,
+    validateCustomAnimationSchema,
+} = await import('../shared/js/schema.js');
+const { initAnimationDB, saveAnimationBlob } = await import('../shared/js/idb_storage.js');
```

ESLint

[error] 'initAnimationDB' is assigned a value but never used.

(no-unused-vars)